docs/understanding-tracking-design/create-tracking-plan-with-tracking-scenarios/managing-tracking-scenarios-console/index.md

```diff
@@ -0,0 +1,166 @@
+
+![](images/ts-1.png)
+
+Next, click the 'Add new scenario' button. A new dialog box will appear with fields for basic information that can be defined during creation, as shown.
+
+![](images/ts-2.png)
+
+Tracking scenarios should be created in association with the most recent version of the data structure deployed in production. This will determine the rules for tracking implementation that can be set in the validation part of the tracking scenarios.
+
+It is always guaranteed that your tracking scenario will have rules compatible with the associated data structure version. This ensures that any event tracked by following the rules defined in the tracking scenarios will pass validation for the associated data structure.
+
+However, if a newer version of the data structure is subsequently deployed in production, you may wish to "upgrade" the tracking scenario and tracking to the newer data structure and verify that your implementation rules still meet validation.
+
+The fields available for filling at this stage are:
+
+- A text field for the **scenario name**
+- A tag field to indicate which applications this scenario is designed to track.
+- A free text field for a **description** of the tracking scenario.
+
+![](images/ts-3.png)
+
+To create a tracking scenario, click the **'Save'** button. If you want to create multiple scenarios at once, such as different scenarios for mobile and web tracking, click the **'Save & add another'** button. You can cancel the creation process by clicking the **'Cancel'** button.
+
+After creating your first tracking scenario, a list of tracking scenarios will appear on the page.
+
+![](images/ts-4.png)
+
+Until published, the scenario status is set to "Draft" to represent that it is a work in progress. During this time, you may wish to share it with others in the business to get feedback or approval. To do this, select the link to the tracking scenario and click the "Share" button located at the top right of the page. Please note that others will need access to the Snowplow Console Data Structures section to view the scenario.
+
+* * *
+
+## Adding more details and editing a Tracking Scenario
+
+To add more information or modify a tracking scenario, select it from the appropriate list view in the "Data Structure and Tracking Scenarios" section. This will open a new page displaying the details of the selected tracking scenario.
+
+![](images/ts-5.png)
+
+To make changes, edit the information in each section, and then click the ‘Save and Update’ button to save your changes.
+
+
+### Implementation rules
+
+The Implementation Rules section is where you can specify how each property for this event should be populated.
+
+To add multiple implementation rules, click the **'+ Add implementation rules'** button in the Implementation Rules section. This will open a dialog view.
+
+The dialog displays the list of properties defined in the data structure associated with this scenario.
+
+![](images/ts-7.png)
```

Comment:
Can we create a more generic product, rather than a car? also this is weird because a car is best suited to be an entity rather than an event.

Reply:
What would be a good candidate for this? Can you think of any good fit on Next environment or that we can create for this purpose? And in that case, should we redo all screenshots with the new one or only for `ts-7.png` screenshot?

Reply:
When I did the docs and demo for Tracking Catalog I created this generic [Add to Basket](https://console.snowplowanalytics.com/782acb70-b482-4906-b223-159f51068f22/tracking-catalog/com.demo.tracking.add_to_basket) event. We could reuse this.

I also created the associated [Product Entity](https://console.snowplowanalytics.com/782acb70-b482-4906-b223-159f51068f22/tracking-catalog/com.snowplowanalytics.snowplow.ecommerce.product).

Reply:
Updated screenshots with Add to Basket event.